lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -312,12 +312,16 @@ public function delete_item( $request ) {
 			}
 		}
 
+		$mock_request = new WP_REST_Request( 'GET', '/wp/users/' . $id );
```

Comment:
Oh. Because it's not routing through the server, the request can be whatever. I guess ultimately it should route through the server, so it's not hard-coupled to this controller.